### PR TITLE
Allow custom media to reference each other

### PR DESCRIPTION
**Pull request by hgl**
The final missing piece. :)

Any plan on settling the option name? I plan to submit the extract pr once it's resolved.

### CODE REVIEW

**Comment by MoOx** [comment]
looks good.
I will go for "computed-definitions" for the name.

File: test/fixtures/append.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@custom-media --a (--a);
```
Comment (MoOx) [inline test/fixtures/append.css]: this fixture seems not used.

**Comment by MoOx** [comment]
Please remove the unused fixtures and rebase on master so I can merge.

File: test/fixtures/append.css (hunk 1 above)
Comment (hgl) [inline test/fixtures/append.css]: It's used by the "should transform custom media with self reference" test.

**Comment by hgl** [comment]
You should be able to merge this PR directly. Let me know if anything conflicts.

File: test/fixtures/append.css (hunk 1 above)
Comment (MoOx) [inline test/fixtures/append.css]: Sorry I don't see how. No import or something in ` test/fixtures/transform-self-reference.css `

Comment (hgl) [inline test/fixtures/append.css]: Oh, you are right. It's been awhile and I have no idea why this file exists. Let me double check.

**Comment by hgl** [comment]
PTAL

**Comment by MoOx** [comment]
Nice.

**Comment by MoOx** [comment]
Released as 4.1.0

**Comment by hgl** [comment]
Thanks. I will send the cssnext PR soon.

**Comment by tbck** [comment]
This PR fixes #9.

**Pull request merged**